plugins/Login/FormLogin.php

```diff
@@ -19,6 +19,7 @@ class FormLogin extends QuickForm2
 {
     function __construct($id = 'login_form', $method = 'post', $attributes = null, $trackSubmit = false)
     {
+        $attributes = array_merge($attributes ?: [], [ 'action' => '?module=Login' ]);
```

Comment:
Would this need to be `getLoginPluginName()` in case a different login plugin is active? instead of hard coded `Login`

Reply:
👍 makes sense